server.js

```diff
@@ -1,5 +1,69 @@
 // import your node modules
-
+const express = require('express')
+const bodyParser = require('body-parser')
 const db = require('./data/db.js');
 
 // add your server code starting here
+const server = express();
+server.use(bodyParser.json())
+server.use(express.json());
+
+
+server.get('/api/posts',(req,res) => {
+    db.find().then((posts) => {
```

Comment:
`db.find().then()` is a Promise, where `.then` handles the resolved Promise, we need a `.catch` to handle in cases where the Promise is rejected.